code/modules/mechanics/MechanicMadness.dm

```diff
@@ -502,11 +501,24 @@
 	proc/componentSay(var/string)
 		string = trim(sanitize(html_encode(string)))
 		var/maptext = null
-		var/mob/user = usr
-		if (src_exists_inside_user_or_user_storage && !src.storage)
-			maptext = make_chat_maptext(src.owner, "[string]", "color: #FFBF00;", alpha = 255)
+		var/loc = null //Location used for center of all_hearers scan "Probably where you want your text attached to."
```

Comment:
Naming the var `loc` isn't necessarily the best idea as it shaows the eixsting var `loc` on the type. Probably better call it something like `maptext_loc`.

Reply:
Was there initially I believe I will go ahead and fix that.